test/Julja2.jl

```diff
@@ -0,0 +1,219 @@
+using Julja2
+using UUIDs
+using Test
+
+struct Book
+    title::String
+    year::Int32
+    price::Float64
+    map_data
+end
+
+my_book = Book(
+    "Advanced Julia Programming",
+    2024,
+    49.99,
+    Dict("author" => "Julia team", "code_link" => "https://github.com/JuliaLang/julia"),
+)
+
+@testset "Variable Insert Test" begin
+    template = Julja2.Julja2Template(
+        """
+        {{ no_such_variable }}<book>
+        <title>{{ title }}</title>
+        <authors>
+            <author lang="en">John Doe</author>
+            <author lang="es">Juan Peréz</author>
+        </authors>
+        <year>{{ year }}</year>
+        <price>{{ price }}</price>
+        <p>Author is {{ map_data["author"] }}. Link to source code: {{ map_data["code_link"] }}</p>
+        </book>""",
```

Comment:
```suggestion
        </book>
        """,
```
Perhaps a new line is needed here?

Reply:
In test as it is no, new like is not needed. 
Template is: 
`
...
<p>Author is {{ map_data["author"] }}. Link to source code: {{ map_data["code_link"] }}</p>
</book>""",
`

The result must be the same. But I will change template and result and add a new line.